internal/locate/replica_selector_test.go

```diff
@@ -1658,6 +1646,49 @@ func TestReplicaReadAccessPathByMixedAndPreferLeaderCase(t *testing.T) {
 	}
 	s.True(s.runCaseAndCompare(ca))
 	s.changeRegionLeader(1)
+
+	cas := []replicaSelectorAccessPathCase{
+		{
+			reqType:   tikvrpc.CmdGet,
+			readType:  kv.ReplicaReadPreferLeader,
+			staleRead: false,
+			timeout:   0,
+			accessErr: []RegionErrorType{ServerIsBusyErr},
+			expect: &accessPathResult{
+				accessPath: []string{
+					"{addr: store1, replica-read: true, stale-read: false}", // store1 will be marked already slow.
+					"{addr: store2, replica-read: true, stale-read: false}",
+				},
+				respErr:         "",
+				respRegionError: nil,
+				backoffCnt:      0,
+				backoffDetail:   []string{},
+				regionIsValid:   true,
+			},
+			afterRun: func() { /* don't invalid region */ },
+		},
+		{
+			reqType:   tikvrpc.CmdGet,
+			readType:  kv.ReplicaReadPreferLeader,
+			staleRead: false,
+			timeout:   0,
+			accessErr: []RegionErrorType{ServerIsBusyErr, ServerIsBusyErr, ServerIsBusyErr},
+			beforeRun: func() { /* don't resetStoreState */ },
+			expect: &accessPathResult{
+				accessPath: []string{
+					"{addr: store2, replica-read: true, stale-read: false}", // won't try leader in store1, since it is slow.
+					"{addr: store3, replica-read: true, stale-read: false}",
+					"{addr: store1, replica-read: true, stale-read: false}",
```

Comment:
I'm wondering what will be if store2 is also slow. I think the access path shall be `store3 -> store1 -> store2`, but v1 seems don't try store2 after store1.
Is it an advantage of v2?

Reply:
@crazycs520 PTAL

Reply:
Nice catch, It looks like v2 is not compatible with v1 in this case, I'm going to modify v2 to be compatible with v1 and add tests for that.

Reply:
@you06 addressed, and add a test for it, PTAL.